.github/workflows/continous-delivery.patch.yml

```diff
@@ -1,35 +0,0 @@
-name: CD
-
-on:
-  # Only patch the Docker image test jobs
-  pull_request:
-    paths-ignore:
```

Comment:
This patch file is still required because we want the release image jobs to run on PRs. It will also need to be updated with the new set of jobs.

Reply:
Reverted